apps/onnx/onnx_converter_generator_test.cc

```diff
@@ -0,0 +1,31 @@
+#include "HalideBuffer.h"
+#include "HalideRuntime.h"
+#include "test_model.h"
+#include <iostream>
+#include <random>
+
+int main(int argc, char **argv) {
+    std::cout << "Running onnx_converter_generator_test..." << std::endl;
```

Comment:
Nit: Please just use '\n' instead of endl. endl also implies a buffer flush, which is unnecessary, so usually endl is more verbose and slightly slower.